examples/DIPDialect/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 if(BUDDY_ENABLE_OPENCV)
   find_package(OpenCV REQUIRED CONFIG)
```

Comment:
This condition is BUDDY_ENABLE_OPENCV, not enable jpeg or etc. . Can we bind libjpeg, libpng, libbmp and BuddyLibDIP together so that we only need to link one library(or variable such as ${DIP_LIBS})?